tapa/common/graph.py

```diff
@@ -86,3 +93,101 @@ def get_flatten_graph(self) -> "Graph":
         }
 
         return Graph(self.name, new_obj)
+
+    def get_floorplan_slot(
+        self, slot_name: str, tasks_in_slot: list[str], top: Base
```

Comment:
Suggest renaming `tasks_in_slot` -> `instances_in_slot`

A task can have multiple instances and only some instances are in the slot.